dts/arm/st/stm32f4-pinctrl.dtsi

```diff
@@ -27,6 +27,12 @@
 					tx = <STM32_PIN_PB6 (STM32_PINMUX_ALT_FUNC_7 | STM32_PUSHPULL_NOPULL)>;
 				};
 			};
+			usart1_pins_d: usart1@3 {
+				rx_tx {
+					rx = <STM32_PIN_PA10 (STM32_PINMUX_ALT_FUNC_7 | STM32_PUSHPULL_PULLUP)>;
```

Comment:
Not sure it is actually required (I've never mucked with the drive settings for UARTs on STM32) but the other definitions are using `STM32_PUPDR_NO_PULL`.

Reply:
True ´- it's a copy and paste error. Still seems to work. ha ha